sources/packages/web/src/views/aest/SupportingUser.vue

```diff
@@ -53,35 +69,17 @@ export default defineComponent({
           props.supportingUserId,
         );
       formName.value = supportingUsersData.formName;
-      let contactAddress = {};
-      // Here there is only one address for now
-      if (supportingUsersData.contactInfo?.address) {
-        const address = supportingUsersData.contactInfo.address;
-        contactAddress = {
-          city: address.city,
-          country: address.country,
-          provinceState: address.provinceState,
-          postalCode: address.postalCode,
-          addressLine1: address.addressLine1,
-          addressLine2: address.addressLine2,
-        };
-      }
-      // for both parent and partner first tab is same
-      // and the information on the 2nd tab is fed in `supportingData`
       formData.value = {
-        isAbleToReport: supportingUsersData.isAbleToReport,
-        givenNames: supportingUsersData.firstName,
-        lastName: supportingUsersData.lastName,
-        email: supportingUsersData.email,
-        dateOfBirth: dateOnlyLongString(supportingUsersData.birthDate),
-        sin: supportingUsersData.sin,
-        phone: supportingUsersData.contactInfo?.phone,
         supportingData: supportingUsersData.supportingData,
-        ...contactAddress,
-        hasValidSIN: supportingUsersData.hasValidSIN,
+        parentFullName: supportingUsersData.parentFullName,
```

Comment:
`parentFullName` is the only required value from the API in this component.
Can you please remove the `formName` and `formData`, can create a single reactive prop for the `parentFullName`?